### DISCUSSION TITLE
non-Standard InChI/InChIKey for SMILES with dummy atoms (IMPORTANT)

### DISCUSSION THREAD

**Discussion by Lili-Summer** [rdkit/rdkit#7616] [Q&A]
Hello,

I have two questions:

1- How to print the version of InChI software being supported in RDKIt 
2- How to calculate the non-standard InChI/InChIKey for SMILES with dummy atoms
 (e.g. *.CS(=O)(=O)NCc1ccc(N2C[C@H](NC(=O)c3nnc(CCc4ccc(Cl)cc4)s3)CC2=O)cc1)

I am registering a series of molecules, and I am facing challenges with generating non-standard InChI/InChIKey for SMILES containing dummy atoms.

Thank you!
Leila

**Comment by glevans** [comment]
I am also interested in the answer following question:

**How to print the version of InChI software being supported in RDKIt?**

**Reply by greglandrum** [reply]
There is currently no way to do this from within the RDKit itself.
At the moment you would need to go and look in github for the RDKit release you are using and look at the contents of the FindInchi.cmake file to see which version we are working with. Here's an example of one of that URL for the 2025.03.2 release:
https://github.com/rdkit/rdkit/blob/Release_2025_03_2/Code/cmake/Modules/FindInchi.cmake#L42

I think it should be possible for use to provide access to the information from within the RDKit. @glevans if the info would be useful for you, please create a feature request and I can look into it.

**Reply by glevans** [reply]
Thank you so much!

I have made a feature request:
https://github.com/rdkit/rdkit/issues/8508
